types/windows-cpu/index.d.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Public class for WindowsCPU
+ */
+declare class WindowsCPU {
+    /**
+     * Access to uninstantiated WindowsCPU class
+     */
+    static WindowsCPU: { new(): WindowsCPU };
```

Comment:
Per https://unpkg.com/browse/windows-cpu@1.1.0/index.js#L27, this should be an instance property:

```suggestion
    WindowsCPU: typeof WindowsCPU;
```